src/main/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeSummaryRepository.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.blobrouter.data;
+
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.blobrouter.data.model.EnvelopeSummary;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+
+@Repository
+public class EnvelopeSummaryRepository {
+
+    private static final String BULKSCAN_CONTAINER_NAME = "bulkscan";
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+    private final EnvelopeSummaryMapper mapper;
+
+    public EnvelopeSummaryRepository(
+        NamedParameterJdbcTemplate jdbcTemplate,
+        EnvelopeSummaryMapper mapper
+    ) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.mapper = mapper;
+    }
+
+    public List<EnvelopeSummary> find(Instant from, Instant to) {
+        return jdbcTemplate.query(
+              "SELECT container, file_name, file_created_at, dispatched_at, "
+                + "       status, is_deleted "
+                + "FROM envelopes "
+                + "WHERE container <> '" + BULKSCAN_CONTAINER_NAME + "' "
```

Comment:
so still no to moving to named parameter?

Reply:
Do we event need a new repository with new model and new query.
Can we just read all envelopes and filter out the ones from `bulk_scan` container in service layer??

Reply:
what if report queries get nastier? and in real life they should be - complete all in query as it's fast and respond with csv or whatever. would be very wasteful to get data and then filter/amend data in service layer.

since we know it's report, i would vote to rename repository to `ReportRepository` to keep all those in there and each method will represent specific report

Reply:
Will rename to ReportRepository
And having report related stuff in a separate repo makes it future proof in case of new requirements

Reply:
ok